frontend/src/components/transactions/ActionMessage.tsx

```diff
@@ -51,16 +51,8 @@ const transactionMessageRenderers: TransactionMessageRenderers = {
     actionArgs
   }: Props<T.FunctionCall>) => {
     const args = Buffer.from(actionArgs.args, "base64").toString();
-    let parameter;
-    if (args === "{}") {
-      parameter = "";
-    } else {
-      const argums = args.slice(1, args.length - 1).split(",");
-      parameter = argums.map(ar => {
-        const index = ar.indexOf(":");
-        return ar.slice(0, index);
-      });
-    }
+    const args_object = JSON.parse(args);
+    let parameter = Object.keys(args_object);
```

Comment:
This is not related to the issue, right? We address it separately, so revert these changes for this PR.